keps/0001-kubernetes-enhancement-proposal-process.md

```diff
@@ -33,110 +33,109 @@ status: implemented
   Kubernauts
-- support the creation of _high value user facing_ information such as:
+- support the creation of _high-value, user-facing_ information such as:
   - an overall project development roadmap
-  - motivation for impactful user facing changes
-- reserve GitHub issues for tracking work in flight rather than creating "umbrella"
+  - motivation for impactful user-facing changes
+- reserve GitHub issues for tracking work in flight, instead of creating "umbrella"
   issues
-- ensure community participants are successfully able to drive changes to
+- ensure community participants can successfully drive changes to
   completion across one or more releases while stakeholders are adequately
   represented throughout the process
 
-This process is supported by a unit of work called a Kubernetes Enhancement Proposal or KEP.
-A KEP attempts to combine aspects of a
+This process is supported by a unit of work called a Kubernetes Enhancement Proposal, or KEP.
+A KEP attempts to combine aspects of
 
-- feature, and effort tracking document
+- a feature, and effort-tracking document
 - a product requirements document
-- design document
+- a design document
 
-into one file which is created incrementally in collaboration with one or more
+into one file, which is created incrementally in collaboration with one or more
 Special Interest Groups (SIGs).
 
 ## Motivation
 
-For cross project SIGs such as SIG PM and SIG Release an abstraction beyond a
-single GitHub Issue or Pull request seems to be required in order to understand
-and communicate upcoming changes to Kubernetes.  In a blog post describing the
-[road to Go 2][], Russ Cox explains
+For cross-project SIGs such as SIG Release, an abstraction beyond a
+single GitHub Issue or pull request seems to be required in order to understand
+and communicate upcoming changes to Kubernetes. In a blog post describing the
+[road to Go 2][], Russ Cox explains:
 
 > that it is difficult but essential to describe the significance of a problem
 > in a way that someone working in a different environment can understand
 
-as a project it is vital to be able to track the chain of custody for a proposed
+It is vital for the project to be able to track the chain of custody for a proposed
 enhancement from conception through implementation.
 
-Without a standardized mechanism for describing important enhancements our
+Without a standardized mechanism for describing important enhancements, our
 talented technical writers and product managers struggle to weave a coherent
-narrative explaining why a particular release is important. Additionally for
-critical infrastructure such as Kubernetes adopters need a forward looking road
-map in order to plan their adoption strategy.
+narrative explaining why a particular release is important. Additionally, adopters of
+critical infrastructure such as Kubernetes need a forward-looking roadmap in order to plan their adoption strategies.
```

Comment:
```suggestion
narrative explaining why a particular release is important. Additionally, adopters
of critical infrastructure such as Kubernetes need a forward-looking roadmap in
order to plan their adoption strategies.
```
nit: line length  - This applies to several changes throughout the doc.

We generally prefer keeping things around 80 characters. It's not necessarily something we require, but something we aim for as they make side-by-side reviews easiers. 

It's also less enforced for docs that weren't already formatted with it in mind, but when making changes to one that was; we try and maintain it 👍 

ref: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#line-length

Reply:
Ah, OK -- this wasn't apparent when I edited the issue triage page :)